test/functional/example_test.py

```diff
@@ -180,7 +175,7 @@ def run_test(self):
             block.solve()
             block_message = msg_block(block)
             # Send message is used to send a P2P message to the node over our NodeConn connection
-            node0.send_message(block_message)
+            self.nodes[0].send_message(block_message)
```

Comment:
I've left the changes to example_test.py in the first commit, since I think it's good to demonstrate the usage of the new functionality in the same commit as the functionality being added.

However, I have changed these calls to be `node.p2p.send_message()` as suggested below.